libutils/include/utils/builtin.h

```diff
@@ -29,6 +31,18 @@
 #define BSWAP_WORD(x) __builtin_bswap64(x)
 #endif
 
+#if CONFIG_WORD_SIZE == 32
+#define CTZ_WORD(x) CTZL(x)
+#elif CONFIG_WORD_SIZE == 64
+#define CTZ_WORD(x) CTZLL(x)
```

Comment:
Actually, on 64-bit systems, it is quite common that `long` is 64-bit. So we could also go with `#define CTZ_WORD(x) CTZL()` in both cases. Note that In `libutils/include/utils/arith.h` this lib already uses `CTZL()`.

Reply:
You are right, but I tested it using both x64 and arm64 `msvc` 19.30 and get that `long` is 32bits size. So I suggest that we use the more likely selection for `word` size type.

Reply:
Oh - if `long` is 32-bit on a 64-bit architecture, I'd expect this to break a lot more things...

Reply:
`long` is definitely 64 on the compilers that are supported for the 64 bit architectures that are supported.